Healthy/Classes/Extensions/UILabel+Style.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  UILabel+Style.swift
+//  Healthy
+//
+//  Created by Abdallah on 14/05/2023.
+//
+
+import UIKit
+
+extension UILabel{
+    
+    // MARK:  Stryle tupe
+    
```

Comment:
There is a spelling mistake in the comment for the "Style Type" section (Stryle tupe). It should be corrected to Style Type for clarity.